video/out/vulkan/context.c

```diff
@@ -189,17 +189,30 @@ pl_vulkan mppl_create_vulkan(struct vulkan_opts *opts,
 #ifdef VK_EXT_SHADER_OBJECT_EXTENSION_NAME
         VK_EXT_SHADER_OBJECT_EXTENSION_NAME,
 #endif
+        VK_KHR_COOPERATIVE_MATRIX_EXTENSION_NAME,
         VK_KHR_VIDEO_DECODE_QUEUE_EXTENSION_NAME,
         VK_KHR_VIDEO_DECODE_H264_EXTENSION_NAME,
         VK_KHR_VIDEO_DECODE_H265_EXTENSION_NAME,
         VK_KHR_VIDEO_QUEUE_EXTENSION_NAME,
+        VK_KHR_VIDEO_MAINTENANCE_1_EXTENSION_NAME,
+        VK_KHR_SHADER_RELAXED_EXTENDED_INSTRUCTION_EXTENSION_NAME,
         "VK_KHR_video_decode_av1", /* VK_KHR_VIDEO_DECODE_AV1_EXTENSION_NAME */
```

Comment:
If you are syncing this, why vp9 is not added?

Reply:
Because I forgot about it. I had put it aside because my local headers aren't new enough and then didn't go back to it. I will add.